docs/protocol/extension_statistics.md

```diff
@@ -111,8 +112,16 @@ $model_stat =
   DOES NOT include cache hits.
 
 - "inference_stats" : The aggregate statistics for the
-  model/version. So, for example, "inference_stats":"success"
-  indicates the number of successful inference requests for the model.
+  model. So, for example, "inference_stats":"success" indicates the number of
+  successful inference requests for the model.
+
+- "response_stats" : The aggregate decoupled response statistics for the
+  model. For example, { "key" : { "response_stats" : "success" } } indicates the
+  aggregate statistics of successful decoupled responses at "key" for the model,
+  where "key" identifies each decoupled response generated by the model across
+  different requests. For example, given a model that generates three responses,
```

Comment:
I think it makes sense to call out the example model "decoupled' in this case since it is described to have multiple responses
```suggestion
  different requests. For example, given a decoupled model that generates three responses,
```